examples/cql-examples.json

```diff
@@ -65,6 +65,10 @@
     {
       "name": "bin_maxbins",
       "title": "Histogram - maxbins?"
+    },
+    {
+      "name": "scaleType",
```

Comment:
I see +1  but I haven't seen any change regarding this.  

Also apologies for my typo

`scaleType` => `scale-type`